### ISSUE TITLE
Switch BlenderFDS off again

### ISSUE THREAD

**Issue by TristanHehnen** [firetools/blenderfds#14]
Hi Emanuele,

I've started playing with BlenderFDS. I need some time to get used to it, apart from that I quite like it! In the beginning I followed the tutorial and also hit the `File > Load BlenderFDS Settings` to "make it permanent". However, now I would like to use the regular Blender again.

How do I "switch it off", temporarily? 
Or better: how to activate BlenderFDS temporarily. 

Cheers,
Tristan

**Comment by emanuelegissi** [comment]
Hi,
happy to know you are playing with BlenderFDS.
To switch it off just disable the add-on, restart Blender and File > Reload
default settings
Let me know if it works ok
Emanuele

On Aug 5, 2018 20:36, "FireTristan" <notifications@github.com> wrote:

Hi Emanuele,

I've started playing with BlenderFDS. I need some time to get used to it,
apart from that I quite like it! In the beginning I followed the tutorial
and also hit the File > Load BlenderFDS Settings to "make it permanent".
However, now I would like to use the regular Blender again.

How do I "switch it off", temporarily?
Or better: how to activate BlenderFDS temporarily.

Cheers,
Tristan

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/firetools/blenderfds/issues/14>, or mute the thread
